configure.ac

```diff
@@ -32,6 +32,7 @@ esac
 AC_CHECK_TOOL([WINDRES], [windres], [])
 AS_IF([test "x$build_windows" = "xyes" -a -n "x$WINDRES"], [
 	enable_windows_resource=yes
+	AC_DEFINE([WINDOWS_BUILD], [1], [Add a DEFINE that this is a Windows build])
```

Comment:
It would surprise me very much if autotools do not already have a solution for that.

Reply:
Good point, thank you!

I haven't found a more elegant way to accomplish this and I found this solution to have a big benefit:  `WINDOWS_BUILD` will be defined if and only if the rest of the build will also be made Windows-compatible.

E.g. a solution like https://stackoverflow.com/a/38899152 would have one part of the build depend on `AC_CHECK_TOOL([WINDRES], [windres], [])` and another part of the build depend on `${host_os}`.